examples/gno.land/r/sys/params/params_test.gno

```diff
@@ -1,6 +1,10 @@
 package params
 
-import "testing"
+import (
+	"testing"
+
+	_ "gno.land/r/gov/dao/loader" // so that loader.init is executed
```

Comment:
This is the exact reason why I don't like magical init functions -- this is a hack that's not explicit